src/test/java/com/loofah/graph/api/repositories/SkillsRepositoryTest.java

```diff
@@ -0,0 +1,43 @@
+package com.loofah.graph.api.repositories;
+
+import com.loofah.graph.api.models.Skill;
+import com.loofah.graph.api.testConfig.GraphAPITestConfiguration;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+
+@RunWith(SpringRunner.class)
+@ContextConfiguration(classes = GraphAPITestConfiguration.class)
+public class SkillsRepositoryTest {
+
+    @Autowired
+    private List<Skill> setUpTestSkills;
+
+    private SkillsRepository skillsRepository;
+
+    @Before
+    public void setUp() {
+        skillsRepository = new SkillsRepository(setUpTestSkills);
+    }
+
+    @Test
+    public void gets_all_skills() {
```

Comment:
Have used a different naming convention based on an article I read recently which made quite a lot of sense. 
https://enterprisecraftsmanship.com/posts/you-naming-tests-wrong/